tasks/config/run.js

```diff
@@ -223,6 +223,12 @@ module.exports = function (grunt) {
       args: ['scripts/test_hardening.js'],
     }),
 
+    test_package_safer_lodash_set: scriptWithGithubChecks({
```

Comment:
__question:__ I guess we need to mention this task in Jenkins file?

Reply:
Doh! My bad 😅

Reply:
Fixed 👍